### PR TITLE
Remove the "focus" external component, use client_proxy instead.

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

**Review by damencho — APPROVED** [review]

File: debian/jitsi-meet-prosody.postinst [hunk 1]
```diff
@@ -142,6 +142,21 @@ case "$1" in
...
+        # New:
+        # Component "focus.jitmeet.example.com" "client_proxy"
+        #    target_address = "focus@auth.jitmeet.example.com"
+        if grep -q "Component \"focus.$JVB_HOSTNAME\"" $PROSODY_HOST_CONFIG && ! grep "Component \"focus.$JVB_HOSTNAME\" \"client_proxy\"" $PROSODY_HOST_CONFIG ;then
+            sed -i -e "s/Component \"focus.$JVB_HOSTNAME\"/Component \"focus.$JVB_HOSTNAME\" \"client_proxy\"\n    target_address = \"$JICOFO_AUTH_USER@auth.$JVB_HOSTNAME\"/" $PROSODY_HOST_CONFIG
+            PROSODY_CONFIG_PRESENT="false"
+        fi
+
+        # Make sure the focus@auth user's roster includes the proxy component (this is idempotent)
+        prosodyctl mod_roster_command subscribe focus.$JVB_HOSTNAME $JICOFO_AUTH_USER@auth.$JVB_HOSTNAME
```
Comment (damencho) [inline debian/jitsi-meet-prosody.postinst:158]: One question, though. Will this work with prosody 0.10?

Comment (bgrozev) [reply debian/jitsi-meet-prosody.postinst:158]: I haven't tested. The docs mention it works on 0.8 and 0.9, and it works on 0.11 for what we do, so I expect it also works on 0.10.

Comment (damencho) [reply debian/jitsi-meet-prosody.postinst:158]: If the docs say so, should be fine. I wait for the time to get rid of thinking about 0.10 ... :)

**Comment by paulmenzel** [comment]
The update fails on my system: https://github.com/jitsi/jitsi-meet/issues/8388

**Pull request merged**